packages/manager/apps/sap-features-hub/src/data/hooks/useInstallationHistory.ts

```diff
@@ -16,15 +16,25 @@ export default function useInstallationHistory() {
     queries:
       services?.map(({ serviceName }) => ({
         queryKey: ['sap-features-hub', `/dedicatedCloud/sap`, serviceName],
-        queryFn: () => getSAPInstallations(serviceName),
+        queryFn: async () => {
+          const res = await getSAPInstallations(serviceName);
```

Comment:
May be use queryClient.fetchQuery ?

https://tanstack.com/query/v4/docs/reference/QueryClient#queryclientfetchquery